openapi.yaml

```diff
@@ -140,7 +180,25 @@ components:
         - asyncapi
       properties:
         asyncapi:
-          $ref: '#/components/schemas/AsyncAPIDocument'
+          $ref: "#/components/schemas/AsyncAPIDocument"
+    BundleDocument:
+      type: object
+      required:
+        - asyncapi
+      properties:
+        asyncapi:
+          $ref: "#/components/schemas/AsyncAPIDocument"
```

Comment:
Have in mind that `bundler` package has options that we should described them here. Also you can pass several documents, not only one.

Reply:
Is options are required or optional?
I checked the documentation of the bundler but did not get it there.